cmd/plugins/juju-metadata/toolsmetadata.go

```diff
@@ -118,7 +118,7 @@ func (c *toolsMetadataCommand) Run(context *cmd.Context) error {
 		if err != nil {
 			return errors.Trace(err)
 		}
-		toolsList, err = envtools.FindToolsForCloud(toolsDataSources(source), simplestreams.CloudSpec{}, c.stream, -1, -1, coretools.Filter{})
+		toolsList, err = envtools.FindToolsForCloud(toolsDataSources(source), simplestreams.CloudSpec{}, []string{c.stream}, -1, -1, coretools.Filter{})
```

Comment:
this should have the same behaviour as juju - so it needs to use the streams fallbacks based on what the user specifies

Reply:
Hang on, won't that mean that if they pass `--stream=devel` to the command the resulting metadata it writes out will have everything from devel, proposed and released as if it was in devel? That seems wrong. If someone wants to generate metadata for all streams they can run the tool for each stream. Or should we do the looping over streams here and merge each stream's tools into the metadata separately?